docs/static/managing-multiline-events.asciidoc

```diff
@@ -5,10 +5,17 @@ Several use cases generate events that span multiple lines of text. In order to
 Logstash needs to know how to tell which lines are part of a single event.
 
 Multiline event processing is complex and relies on proper event ordering. The best way to guarantee ordered log
-processing is to implement the processing as early in the pipeline as possible. The preferred tool in the Logstash
-pipeline is the {logstash}plugins-codecs-multiline.html[multiline codec], which merges lines from a single input using
-a simple set of rules.
+processing is to implement the processing as early in the pipeline as possible.
+
+IMPORTANT: If you are using a Logstash input plugin that supports multiple hosts, such as
+the <<plugins-inputs-beats>> input plugin, you should not use the
+<<plugins-codecs-multiline>> codec to handle multiline events. Doing so may result in the 
+mixing of strings and corrupted event data. In this situation, you need to handle multiline
+events before sending the event data to Logstash. 
 
+The <<plugins-codecs-multiline>> codec is the preferred tool for handling multiline events
+in the Logstash pipeline. The multiline codec merges lines from a single input using
```

Comment:
Yup, when formatted correctly in the HTML, it's definitely more clear that this sentence doesn't contradict what users see in the note box. I can see where it wouldn't be as obvious if you're reading the text without formatting.